project/src/test/java/com/google/sps/data/GeoContextTest.java

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import static org.junit.Assert.assertEquals;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+
+@RunWith(JUnit4.class)
+public class GeoContextTest {
+
+    @Test
+    public void getGeoApiContext_followingCalls_returnSameGeoApiContext() {
+      assertEquals(GeoContext.getGeoApiContext(), GeoContext.getGeoApiContext());
```

Comment:
What if the returned object is null?

Reply:
Then it would be null on both calls and the test passes (there is no reason for it ever to be null, but lets say there is some kind of problem with the build that returns null - the subsequent call will result the same, and if not then it's fine the test doen't pass but it should never happen).

Reply:
This is what I meant. If your creation is problematic, this test effectively tests nothing.

Reply:
Testing non-null should suffice here to avoid this problem.

Reply:
Ohhh yes you are right. Added the non-null assertion